testing_dir/minesweeper.py

```diff
@@ -89,7 +89,7 @@ def game_loop():
             player_pos = new_pos
             moves += 1
 
-        if moves == BOARD_SIZE * BOARD_SIZE - NUM_MINES:
+        if moves == BOARD_SIZE * BOARD_SIZE - NUM_MINES + 1:  # Error: should not have +1
```

Comment:
The condition for winning the game seems incorrect. The logic should not include `+1` as it would prematurely end the game.
```diff
-        if moves == BOARD_SIZE * BOARD_SIZE - NUM_MINES + 1:
+        if moves == BOARD_SIZE * BOARD_SIZE - NUM_MINES:
```